### DISCUSSION TITLE
Is `--manifest-generator` supported for LambdaInvoke operations via `aws s3control create-job ...`?

### DISCUSSION THREAD

**Discussion by purplehat7** [aws/aws-cli#7566] [Q&A] [answered]
[It would appear not](https://github.com/aws/aws-cli/issues/7240), but is this expected behavior? Is there a plan to support `--manifest-generator` for `LambdaInvoke` operations at some point?

**Comment by tim-finnigan** [comment, marked as answer]
Hi @purplehat7 thanks for reaching out. In the [create-job documentation](https://awscli.amazonaws.com/v2/documentation/api/latest/reference/s3control/create-job.html) there is a `--manifest-generator` documented. If you update to the latest version of the AWS CLI, are you able to use it? Please let us know if you are running into any issues.
